test/integration/merkle-streamer/ll/claim/claim.t.sol

```diff
@@ -127,15 +126,21 @@ contract Claim_Integration_Test is MerkleStreamer_Integration_Test {
         givenIncludedInMerkleTree
         givenProtocolFeeZero
     {
+        test_Claim(ud(0));
+    }
+
+    function test_Claim(UD60x18 protocolFee) internal {
         uint256 expectedStreamId = lockupLinear.nextStreamId();
 
+        uint128 feeAmount = uint128(ud(defaults.CLAIM_AMOUNT()).mul(protocolFee).intoUint256());
```

Comment:
Let's remove the end line before this declaration

Reply:
I think this line can be removed completely, as my feedback left on `src`

Reply:
yes you're right

Reply:
If we remove this, then the following test will not pass. Below test is necessary to test claim when `protocolFee` is non-zero and in that case amount deposited in the stream equals `claimAmount - feeAmount `.

```solidity

    function test_Claim_ProtocolFeeNotZero()
        external
        givenCampaignNotExpired
        givenNotClaimed
        givenIncludedInMerkleTree
    {
        changePrank({ msgSender: users.admin });
        UD60x18 protocolFee = ud(0.1e18);
        comptroller.setProtocolFee({ asset: asset, newProtocolFee: protocolFee });

        test_Claim(protocolFee);
    }
```

Reply:
ooh, right. you're right @smol-ninja.

Reply:
Thanks for confirming Paul. Do you also agree @andreivladbrg?

Reply:
hmm, i am not yet sure why we need to test both versions:
- when the protocol fee = 0
- when the protocol fee > 0

isn't this already tested in the create functions in `v2-core` ?

my views to this is like if we were to test the `ERC20` balances. we are not doing this because that's a matter of `ERC20` contract tests

Reply:
What you said is true in the purest technical sense but business-wise, it's a good decision to test our own system (ERC20 is different - we haven't written that ourselves).

System A might work by itself, system B might work by itself too, but when you couple A to B, things could go wrong.

so this is basically an end-to-end test and I'm in favor of keeping it just like @smol-ninja has written it.

Reply:
ok got it